### ISSUE TITLE
Testing Lists Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4633]
# Page to Test:

Home page - https://openlibrary.org/lists

# Steps

Steps for this task are outlined in #4521 

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by bpmcneilly** [comment]
@Yashs911 - to keep the large 'Super Task' issues clear, I'm creating individual issues for each page. Just saw your comment about this page there, let's discuss here! 😃

> @bpmcneilly I went through the list page and I noticed that the cover image for a list is a book cover of the book present in that list but the alt attribute says `alt="Cover of: integrity books(list name)"` which is technically wrong.
> ![image](https://user-images.githubusercontent.com/64412143/108536865-f3106980-7302-11eb-9e61-169c2346da37.png)
> ![image](https://user-images.githubusercontent.com/64412143/108536976-120efb80-7303-11eb-99ec-29935a239434.png)

In this instance, the alt text needs to inform users where they will go if they activate the link (See [Example 3 on this page about alt text](https://webaim.org/techniques/alttext/).

In this instance, I think making the alt just "Body" (or whatever the name of the book is) would be most meaningful. Happy to discuss more!

**Comment by Yashs911** [comment]
@bpmcneilly when the user activates the link (by clicking the image or the Title) it will open that particular list. In the above case list name is "Body" and the alt attribute is `alt="Cover of: Body"` so should I only change is to `alt="Body"` or `alt="List: Body"`

**Comment by bpmcneilly** [comment]
@Yashs911 in this instance, I think what makes the most sense would be to simply combine the links for the image & the title of the list. Something like this:
```
<a href="/people/lavapra/lists/OL193921L" class="title results">
  <div class="cover">
    <img src="//covers.openlibrary.org/b/id/777798-S.jpg" alt="">
  </div>
  Historical/faux Historical
</a>
```

Ultimately, I think this link just adds more click areas, but doesn't add any unique content on this page. With the proposed code we've now trimmed one tab stop off of the page by reducing the number of links, & preserved the covers as clickable. Now that the `img` is inside another link, the `alt` can be null, and we've solved the image naming issue. What do you think of that solution?

I think we can delete the `title` attribute unless we're adding any unique content to it (right now it looks like ti was just the same as the alt anyway) - even then, the `title` attribute is unpredictably exposed to ATs.

**Comment by Yashs911** [comment]
@bpmcneilly Sorry for the delay. I will create a PR as soon as I am done with my exams (by the end of next week) :)
